collab-document/tests/document/redo_undo_test.rs

```diff
@@ -0,0 +1,201 @@
+use nanoid::nanoid;
+use serde_json::to_value;
+use std::collections::HashMap;
+use std::time::Duration;
+
+use crate::util::{
+  create_document, create_document_with_db, db, delete_block, insert_block_for_page,
+  open_document_with_db, update_block,
+};
+
+const WAIT_TIME: Duration = Duration::from_secs(1);
+
+#[tokio::test]
+async fn insert_undo_redo() {
+  let doc_id = "1";
+  let test = create_document(1, doc_id);
+  tokio::time::sleep(Duration::from_secs(1)).await;
```

Comment:
```suggestion
  tokio::time::sleep(WAIT_TIME).await;
```

Reply:
Also, this line should be removed because the `enable_undo_redo` is called after create document. So no need to wait 1 sec.